centml/values.yaml

```diff
@@ -1,20 +1,34 @@
 nodeSelector:
   region: LGA1
 
+affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+        - matchExpressions:
+            - key: topology.kubernetes.io/region
+              operator: In
+              values:
+                - "{{.Values.nodeSelector.region}}"
+            - key: gpu.nvidia.com/class
+              operator: In
+              values:
+                - A40
+                - A100_PCIE_40GB
```

Comment:
Please just use a single default GPU type in the values and let the user change it if they would like. No user would randomly want to choose between an A40 and the two A100 types,

Reply:
Removed both A100 GPUs